keyboards/geistmaschine/geist/rules.mk

```diff
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes
```

Comment:
Encoder map can't be enabled at the keyboard level, as it relies on keymap specific things. This should be moved to `keymaps/*/rules.mk`.
```suggestion
# This file intentionally left blank
```